### ISSUE TITLE
hc decks or "tactics" are empty for all civs except austrians, swedish,prussians and duchy of warsaw.Other than that all decks are empty

### ISSUE THREAD

**Issue by MahmoudLCossack** [aoenw/Hundred-Days#34]

**Comment by aoenw** [comment]
Hello @MahmoudLCossack,
Did you perform the step 4 of the [installation](https://github.com/aoenw/Hundred-Days/wiki/Installation-Notes) note?

Usually, if no hc decks are present or empty, it means that the mod's savegame has not been properly installed.

Can you then check the contents of you `My Documents/My Games/` and tell me what you find in it? (I assume you are on windows?)

**Comment by MahmoudLCossack** [comment]
Hi aoenw,
i reinstalled the mod as described in your installation note, i made sure  i merged age of empires 3 folder with the original one in program files and that i put aoe3 hundred days folder in my documents- my games and still  got the same nasty empty deck problem. As i said the prussians, austrians, swedish and polish are working good which means there's nothing wrong with my installation process because all homecity xml files are copy-pasted altogether the same way into my documents my games. i don't know  maybe there's something missing in hc xml files that prevents cards from appearing ingame. i am not a modder so i am not trying to be smart dont get me wrong that's actually why i need your help on this one. plz help.oh and about my windows its windows 7. thanks a lot

**Comment by aoenw** [comment]
Ok, can you tell me the following:
1. What are the names of the deck for austrians?
2. Do you use steam?
3. What are the Age of Empires related folders in My Documents/My Games
4. If you know whether yours standards savegame are stored in My Documents/My Games

**Comment by MahmoudLCossack** [comment]
Okay, there you go
1.Border defense - Mass army- Hungarian regiments 
2.No, I don't
3.just the typical aoe3 folder which contains the folders AI,AI2,scenario, savegame etc and the aoe3 hundred days folder.
4. yes there is a savegame folder in my documents/my games/AOE3 hundred days folder.There is also  a savegame folder in my documents/my games/age of empires 3 but that's related to the original game not the mod.

**Comment by aoenw** [comment]
Can you open one of the savegame xml file for a city which does not work in a web browser and tell me whether the browser displays an error?

Stupid question: you do have the extensions installed, don't you?

I start investigating the differences between the xml files on my side.

**Comment by MahmoudLCossack** [comment]
No there are no errors displayed. I opened all of the xml files though and no errors and if you mean by the extensions the official patches for the game then yes i installed all of them.

**Comment by Yomguithereal** [comment]
It seems that only the civs that does not exist by default in the standard game does not work here.

By extensions I mean _The Warchiefs_ and _The Asian Dynasties_.

Could you copy here the content of, for example, a savegame of a French HC created by your original game?

Other question: is your game in another language than English?

**Comment by MahmoudLCossack** [comment]
yeah I installed the warchiefs and the asian dynasties and they are both patched. My game is in English. Tell me what you're trying to do and maybe i can help you out rather than just giving me questions and then we all can get to the point easier ;). 
Anyways here's a savegame of French HC created by my original game which didn't exist in the first place. I had to play a supremacy game with the french to create one.

<?xml version="1.0" encoding="UTF-16"?>

<savedhomecity version ="2">
    <defaultdirectoryid>0</defaultdirectoryid>
    <defaultfilename>homecityfrench.xml</defaultfilename>
    <civ>French</civ>
    <hctype>HC</hctype>
    <name>Paris</name>
    <heroname>Andre de Condamine</heroname>
    <herodogname></herodogname>
    <shipname></shipname>
    <hcid></hcid>
    <respec>1</respec>
    <level>10</level>
    <xp>141028</xp>
    <skillpoints>9</skillpoints>
    <xppercentage>0.051400</xppercentage>
    <numpropunlocksearned>0</numpropunlocksearned>
    <decks>
        <deck>
            <name>My Deck</name>
            <gameid>4</gameid>
            <default></default>
            <cards>
                <card dbid ="2260">HCShipCoureurs1</card>
                <card dbid ="2572">HCShipFoodCrates1</card>
                <card dbid ="2190">HCShipCoveredWagons</card>
                <card dbid ="1993">HCPioneers</card>
                <card dbid ="2226">HCShipCrossbowmen1</card>
                <card dbid ="2248">HCShipSkirmishers1</card>
                <card dbid ="2249">HCShipSkirmishers2</card>
                <card dbid ="2188">HCShipHussars1</card>
                <card dbid ="2252">HCShipCuirassiers1</card>
                <card dbid ="2253">HCShipCuirassiers2</card>
                <card dbid ="2259">HCNativeCombatTeam</card>
                <card dbid ="2192">HCShipFalconets1</card>
                <card dbid ="2076">HCAdvancedArtillery</card>
                <card dbid ="814">HCUnlockFort</card>
                <card dbid ="2032">HCFurrier</card>
            </cards>
        </deck>
    </decks>
    <activetechs>
        <tech dbid ="2260">HCShipCoureurs1</tech>
        <tech dbid ="2572">HCShipFoodCrates1</tech>
        <tech dbid ="2190">HCShipCoveredWagons</tech>
        <tech dbid ="2226">HCShipCrossbowmen1</tech>
        <tech dbid ="2248">HCShipSkirmishers1</tech>
        <tech dbid ="2249">HCShipSkirmishers2</tech>
        <tech dbid ="2188">HCShipHussars1</tech>
        <tech dbid ="2252">HCShipCuirassiers1</tech>
        <tech dbid ="2253">HCShipCuirassiers2</tech>
        <tech dbid ="2192">HCShipFalconets1</tech>
        <tech dbid ="2076">HCAdvancedArtillery</tech>
        <tech dbid ="2259">HCNativeCombatTeam</tech>
        <tech dbid ="1993">HCPioneers</tech>
        <tech dbid ="814">HCUnlockFort</tech>
        <tech dbid ="2032">HCFurrier</tech>
    </activetechs>
    <activeprops>
        <prop enabled ="1">French_LightSet1</prop>
        <prop enabled ="1">French_AcademyTexture0</prop>
        <prop enabled ="1">French_CathedralTexture0</prop>
        <prop enabled ="1">French_DockTexture0</prop>
        <prop enabled ="1">French_MarketTexture0</prop>
        <prop enabled ="1">French_ManuPlantTexture0</prop>
    </activeprops>
</savedhomecity>

**Comment by MahmoudLCossack** [comment]
<?xml version="1.0" encoding="UTF-16"?>

<savedhomecity version ="2">
	<defaultdirectoryid>0</defaultdirectoryid>
	<defaultfilename>homecityfrench.xml</defaultfilename>
	<civ>French</civ>
	<hctype>HC</hctype>
	<name>Paris</name>
	<heroname>Andre de Condamine</heroname>
	<herodogname></herodogname>
	<shipname></shipname>
	<hcid></hcid>
	<respec>1</respec>
	<level>10</level>
	<xp>141028</xp>
	<skillpoints>9</skillpoints>
	<xppercentage>0.051400</xppercentage>
	<numpropunlocksearned>0</numpropunlocksearned>
	<decks>
		<deck>
			<name>My Deck</name>
			<gameid>4</gameid>
			<default></default>
			<cards>
				<card dbid ="2260">HCShipCoureurs1</card>
				<card dbid ="2572">HCShipFoodCrates1</card>
				<card dbid ="2190">HCShipCoveredWagons</card>
				<card dbid ="1993">HCPioneers</card>
				<card dbid ="2226">HCShipCrossbowmen1</card>
				<card dbid ="2248">HCShipSkirmishers1</card>
				<card dbid ="2249">HCShipSkirmishers2</card>
				<card dbid ="2188">HCShipHussars1</card>
				<card dbid ="2252">HCShipCuirassiers1</card>
				<card dbid ="2253">HCShipCuirassiers2</card>
				<card dbid ="2259">HCNativeCombatTeam</card>
				<card dbid ="2192">HCShipFalconets1</card>
				<card dbid ="2076">HCAdvancedArtillery</card>
				<card dbid ="814">HCUnlockFort</card>
				<card dbid ="2032">HCFurrier</card>
			</cards>
		</deck>
	</decks>
	<activetechs>
		<tech dbid ="2260">HCShipCoureurs1</tech>
		<tech dbid ="2572">HCShipFoodCrates1</tech>
		<tech dbid ="2190">HCShipCoveredWagons</tech>
		<tech dbid ="2226">HCShipCrossbowmen1</tech>
		<tech dbid ="2248">HCShipSkirmishers1</tech>
		<tech dbid ="2249">HCShipSkirmishers2</tech>
		<tech dbid ="2188">HCShipHussars1</tech>
		<tech dbid ="2252">HCShipCuirassiers1</tech>
		<tech dbid ="2253">HCShipCuirassiers2</tech>
		<tech dbid ="2192">HCShipFalconets1</tech>
		<tech dbid ="2076">HCAdvancedArtillery</tech>
		<tech dbid ="2259">HCNativeCombatTeam</tech>
		<tech dbid ="1993">HCPioneers</tech>
		<tech dbid ="814">HCUnlockFort</tech>
		<tech dbid ="2032">HCFurrier</tech>
	</activetechs>
	<activeprops>
		<prop enabled ="1">French_LightSet1</prop>
		<prop enabled ="1">French_AcademyTexture0</prop>
		<prop enabled ="1">French_CathedralTexture0</prop>
		<prop enabled ="1">French_DockTexture0</prop>
		<prop enabled ="1">French_MarketTexture0</prop>
		<prop enabled ="1">French_ManuPlantTexture0</prop>
	</activeprops>
</savedhomecity>

**Comment by MahmoudLCossack** [comment]
sorry that's not all of it

**Comment by MahmoudLCossack** [comment]
?xml version="1.0" encoding="UTF-16"?>

-<savedhomecity version="2">

<defaultdirectoryid>0</defaultdirectoryid>

<defaultfilename>homecityfrench.xml</defaultfilename>

<civ>French</civ>

<hctype>HC</hctype>

<name>Paris</name>

<heroname>Andre de Condamine</heroname>

<herodogname/>

<shipname/>

<hcid/>

<respec>1</respec>

<level>10</level>

<xp>141028</xp>

<skillpoints>9</skillpoints>

<xppercentage>0.051400</xppercentage>

<numpropunlocksearned>0</numpropunlocksearned>


-<decks>


-<deck>

<name>My Deck</name>

<gameid>4</gameid>

<default/>


-<cards>

<card dbid="2260">HCShipCoureurs1</card>

<card dbid="2572">HCShipFoodCrates1</card>

<card dbid="2190">HCShipCoveredWagons</card>

<card dbid="1993">HCPioneers</card>

<card dbid="2226">HCShipCrossbowmen1</card>

<card dbid="2248">HCShipSkirmishers1</card>

<card dbid="2249">HCShipSkirmishers2</card>

<card dbid="2188">HCShipHussars1</card>

<card dbid="2252">HCShipCuirassiers1</card>

<card dbid="2253">HCShipCuirassiers2</card>

<card dbid="2259">HCNativeCombatTeam</card>

<card dbid="2192">HCShipFalconets1</card>

<card dbid="2076">HCAdvancedArtillery</card>

<card dbid="814">HCUnlockFort</card>

<card dbid="2032">HCFurrier</card>

</cards>

</deck>

</decks>


-<activetechs>

<tech dbid="2260">HCShipCoureurs1</tech>

<tech dbid="2572">HCShipFoodCrates1</tech>

<tech dbid="2190">HCShipCoveredWagons</tech>

<tech dbid="2226">HCShipCrossbowmen1</tech>

<tech dbid="2248">HCShipSkirmishers1</tech>

<tech dbid="2249">HCShipSkirmishers2</tech>

<tech dbid="2188">HCShipHussars1</tech>

<tech dbid="2252">HCShipCuirassiers1</tech>

<tech dbid="2253">HCShipCuirassiers2</tech>

<tech dbid="2192">HCShipFalconets1</tech>

<tech dbid="2076">HCAdvancedArtillery</tech>

<tech dbid="2259">HCNativeCombatTeam</tech>

<tech dbid="1993">HCPioneers</tech>

<tech dbid="814">HCUnlockFort</tech>

<tech dbid="2032">HCFurrier</tech>

</activetechs>


-<activeprops>

<prop enabled="1">French_LightSet1</prop>

<prop enabled="1">French_AcademyTexture0</prop>

<prop enabled="1">French_CathedralTexture0</prop>

<prop enabled="1">French_DockTexture0</prop>

<prop enabled="1">French_MarketTexture0</prop>

<prop enabled="1">French_ManuPlantTexture0</prop>

</activeprops>

</savedhomecity>

**Comment by MahmoudLCossack** [comment]
sorry contents of the xml file could not be pasted correctly on this thread. Please tell me what you're planning to do with the file and I will help you.

**Comment by aoenw** [comment]
I am trying to detect structural differences between the main game's file and the mods' ones. But since only the HC concerning the main game's already existing civs are affected, I am suspecting the problem comes from elsewhere. What is weird is this problem never occurred before and I installed the mod on many computers before.

**Comment by MahmoudLCossack** [comment]
I have a question: Do the mod savegame xml files work depending on the original game xml files? meaning that if you don't have for instance the Portuguese savegame xml file for the original game that means the portuguese civ in hundred days deck will be empty???

**Comment by aoenw** [comment]
No, this does not work this way. Is the mod working as expected apart from those nasty empty decks?

**Comment by MahmoudLCossack** [comment]
yes, working great

**Comment by aoenw** [comment]
I am a bit clueless here. Can you check the `AI5` folder and count the number of ai personalities you'll find? Else can you check the contents of the release zip and check whether every file was correctly merged into your AoE3 folder?

**Comment by MahmoudLCossack** [comment]
yes 11 personality files were merged into my aoe3 folder
